packages/react-core/tsconfig.dev.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    // rollup does not allow `declarationDir` to be assigned in the tsconfig
```

Comment:
What does this mean? Looks like this is in the tsconfig?

Reply:
The dev tsconfig extends from the base tsconfig

Reply:
ok, then maybe change the comment to say 
```
 // rollup does not allow `declarationDir` to be assigned in the base tsconfig
 ```